db/users.js

```diff
@@ -1,5 +1,10 @@
 // Functions for manipulating the users table
 const knex = require("./knex");
+const DEFAULT_SETTINGS = {
+  theme: "light",
+  fontSize: "12px",
+  fontFamily: "sans-serif",
```

Comment:
Was there a thought process behind picking sans-serif specifically? Just curious

Reply:
These are just some example settings to show the functionality of the 'settings' value in the database. These will definitely change; new settings will be added and these may be removed.